mavros/src/plugins/imu.cpp

```diff
@@ -433,6 +440,10 @@ class IMUPlugin : public plugin::PluginBase {
 		if (m_uas->is_ardupilotmega()) {
 			accel_ned *= MILLIG_TO_MS2;
 			accel_enu *= MILLIG_TO_MS2;
+
```

Comment:
can you delete this new line please @ChristophTobler?

Reply:
done